ci/playbooks/multinode-customizations.yml

```
@@ -108,16 +108,32 @@
                 validate: "{{ _validate }}"
 
             # Note(Lewis): Only needed for CRC => 2.32.0-4.14.8
+            - name: Wipe listen-address parameters from config to avoid duplication
+              when:
+                - not _dnsmasq.stat.exists
+              become: true
+              ansible.builtin.lineinfile:
+                path: "{{ _dnsmasq_config }}"
+                regexp: '^listen-address'
+                state: absent
+
             - name: Configure dnsmasq listen-address to listen on both br-ex and ci-private-network
               when:
                 - not _dnsmasq.stat.exists
               become: true
               ansible.builtin.lineinfile:
                 path: "{{ _dnsmasq_config }}"
-                insertafter: '^listen-address='
-                line: "listen-address={{ _crc_default_net_ip | ansible.utils.ipaddr('address') }}"
+                insertbefore: '^expand-hosts'
```

Is there a reason it needs to be before expand-hosts? if not, please just add it to EOF like line 95
```suggestion
                insertafter: "EOF"
```